lib/uksched/include/uk/thread.h

```diff
@@ -54,45 +54,74 @@ typedef void (*uk_thread_fn1_t)(void *) __noreturn;
 typedef void (*uk_thread_fn2_t)(void *, void *) __noreturn;
 
 struct uk_thread {
-	struct ukarch_ctx    ctx;	/**< Architecture context */
-	struct ukarch_ectx *ectx;	/**< Extended context (FPU, VPU, ...) */
-	uintptr_t           tlsp;	/**< Current active TLS pointer */
-	__uptr            uktlsp;	/**< Unikraft TLS pointer */
+	/* Architecture context */
+	struct ukarch_ctx    ctx;
+	/* Extended context (FPU, VPU, ...) */
+	struct ukarch_ectx *ectx;
+	/**< Current active TLS pointer */
```

Comment:
```suggestion
	/** Current active TLS pointer */
```
Define what "current" and "active" mean. Mention that TLS stands for _Thread-Local Storage_.